src/http/any-catchall/index.js

```diff
@@ -1,14 +1,15 @@
 let { http } = require('@architect/functions')
+let asap = require('@architect/asap')
 
 // middleware to preserve old urls
 let redirect = require('./redirect')
 
 // middleware proxy s3 assets
-let asap = http.proxy({
+let static = asap({
```

Comment:
Fyi, careful with `static`, it's reserved! https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#future_reserved_keywords